guide/blueprints/effectors.md

```diff
@@ -52,6 +52,114 @@ Here is a simple example of an `SshCommandEffector` definition:
+
+It enables execution of a `command` in a specific container managed by a Kubernetes cluster. _Under the bonnet_ the commands and other configurations are used to generate a Kubernetes job that will execute in its own namespace. Regardless of the job execution result (success or failure) the namespace is deleted at the end, unless configured otherwise. 
+
+There are a number of configuration keys available for the `ContainerEffector`:
+
+| Configuration Key     | Default | Description                                                                                                                                                                                                                           |
+|-----------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| image                 |         | Docker image name, the container will be created from. (mandatory)                                                                                                                                                                    |
+| imagePullPolicy       | `Always` | Possible values: `IfNotPresent`, `Always`, `Never`. Same values from the Kubernetes official documentation, the only advantage is that Apache Brooklyn is case insensitive. So, for example 'NEVER` and `never` are accepted as well. |
+| containerName         |         | The name Kubernetes will give to the container. If not provided, it will be generated from the image name, the entity ID and a random string.                                                                                         |
+| keepContainerForDebug | `false` | When set to true, the namespace" and associated resources and services are not destroyed after execution, thus allowing access to the container for in-dept debugging.                                                                |
+| commands              |         | A list of commands to execute on the container.                                                                                                                                                                                       |
+| args                  |         | If the container is declared with an `ENTRYPOINT`, you might want to provide only arguments for the default command configured by the container.                                                                                      |
+| timeout               | `5m`    | How much should Kubernetes wait before considering a job to be failed and mark the container as failed as well.                                                                                                                       |
```

Comment:
Default is 1 minute.

Reply:
Not for Out effector it is not, because you configured it differently. I will clarify. thx.